README.md

```diff
@@ -60,12 +60,6 @@ check the FIWARE Catalogue entry for
 You can find the User & Programmer's Manual and the Installation &
 Administration Manual on [readthedocs.io](https://fiware-orion.readthedocs.io)
 
-For documentation previous to Orion 0.23.0 please check the manuals at FIWARE
-public wiki:
-
--   [Orion Context Broker - Installation and Administration Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_Installation_and_Administration_Guide)
--   [Orion Context Broker - User and Programmers Guide](https://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/Publish/Subscribe_Broker_-_Orion_Context_Broker_-_User_and_Programmers_Guide)
```

Comment:
This links are very old and no longer working (they redirect to the FIWARE home page), so it's better to remove them

NTC (informative comment)